### DISCUSSION TITLE
Best hardware to get the best experince?

### DISCUSSION THREAD

**Discussion by SD-96** [LizardByte/community#2] [Sunshine]
### Select Topic Area

Question

### Body

I love sunshine and moonlight and have been using them for a very long time. 
But choosing the best hardware to optimize the experience seems very hard. The best encoder, AMD or NVIDIA? I read somewhere that AMD doesn't have b-frame or something, but what kind of important is it? Latency? Quality?   
And let's not forget decoders? I find considerable differences in using Decoders on Mobil devices like Qualcomm and Exynos, compared to my Ryzen 7 7840s CPU (using the built-in statistics).  
It would be nice to get a few examples to squeeze out the best performance possible using Sunshine.  

Again, love the work you guys do 😁

**Comment by ReenigneArcher** [comment]
Sorry for the slow response. This space was not actually used for discussions, but I am planning to migrate each repos space here somewhat soon.

It's really difficult to say what gives the best experience, as there are a lot of factors that go into it besides encoder and decoder. People have success with both AMD and NVIDIA to be honest, but you might have a look through the open issues list.

For mobile devices that's even harder to give suggestions just due to the sheer number of options... Perhaps we need to create some kind of benchmark where users could share results easily.

[a comment by an automated account is not shown]
